sambacc/samba_cmds.py

```diff
@@ -28,6 +28,14 @@
 _GLOBAL_DEBUG: str = ""
 
 
+def get_samba_specifics() -> list:
```

Comment:
Let's return a set here. Most of the time we'd be wanting to check for the presence of a specific entry and I think that fits the set's semantics better. We should also use something like `typing.Set[str]` for our return since we know the members will be strings.

Reply:
Done.